public/index.html

```diff
@@ -47,8 +47,8 @@
                 botConnection,
                 user,
                 bot,
-//              allowMessagesFrom: ['http://localhost:8080']
+//              allowMessagesFrom: ['https://directline.botframework.com/v3/directline'] 
```

Comment:
This is not correct. The purpose of this optional parameter is to allow the experimental "backchannel" feature. It is unrelated to (and would never be) the Direct Line URL.